bin/zopen-init

```diff
@@ -335,7 +382,41 @@ setDefaults()
     releaseLine="STABLE"
     printInfo "- Configured zopen to use stable releaseline packages"
   fi
-  echo "${releaseLine}" > "${rootfs}/etc/zopen/releaseline"
+
+  cat <<zz > "${jsonConfigWorking}"
+  {
+    "release_line": "${releaseLine}",
+    "num_rm_procs": ${RM_FILEPROCS},
+    "is_collecting_stats": ${isCollectingStats}
+  }
+zz
+  jq '.' $jsonConfigWorking > $jsonConfig
+}
+
+generateAnalyticsConfiguration()
+{
+  if ! ${isCollectingStats}; then
+    printVerbose "You have chosen to not collect statistics."
+    return;
+  fi
+  jsonConfig="${rootfs}/etc/zopen/analytics.json"
+  if [ -e "${jsonConfig}" ]; then
+    return;
+  fi
+  jsonConfigWorking="${jsonConfig}.working"
+
+  UUID=$(generateUUID)
```

Comment:
should generateUUID take the host name or something as a parameter so people know how it's being constructed?

Reply:
It's currently defined in https://github.com/ZOSOpenTools/meta/pull/607/files#diff-8953cba62436c52982340b49f55152195f0a6119ee3fab71fba4e8f7d4049a16R1154 using date and $RANDOM. We could include hostname but I wanted to keep it as anonymous as possible for now.